src/vscode-cosmos.api.d.ts

```diff
@@ -0,0 +1,30 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+export interface CosmosDBItem {
+    accountName: string | undefined
+    connectionString: string | undefined
+    cosmosDBTreeItemId: string | undefined
+    databaseName: string | undefined
+}
+
+export interface VSCodeCosmosDB {
+    /**
+     * Founds the database in CosmosDB and return info about it including TreeItem id in CosmosDB Explorer
+     * @param connectionData The connection string and/or other info that provided for successfull database searching
+     */
+    getDatabase(connectionData: CosmosDBItem): Promise<CosmosDBItem | undefined>;
```

Comment:
connectionData should not be of type `CosmosDBItem`. It should be something like `{ connectionString: string }` since that's the only thing you support right now.